go.mod

```diff
@@ -236,4 +235,7 @@ require (
 
 require k8s.io/utils v0.0.0-20230726121419-3b25d923346b // indirect
 
-require github.com/go-ini/ini v1.67.0 // indirect
+require (
+	github.com/go-ini/ini v1.67.0 // indirect
+	k8s.io/gengo/v2 v2.0.0-20240228010128-51d4e06bde70 // indirect
+)
```

Comment:
Is it possible to merge this `require` section with existing one for `// indirect` deps?

Reply:
Sure, done